### ISSUE TITLE
Examples?

### ISSUE THREAD

**Issue by kamakshidasan** [harishd10/contour-tree#2]
I'm trying to compute the Merge Tree for data over here: https://klacansky.com/open-scivis-datasets/

After computation,
the part.raw doesn't seem to open in Paraview and the segmentation always seem to have only one arc.

Could you check this out?

**Comment by harishd10** [comment]
I cannot reproduce the problem. I tried with 2 datasets --- marschner_lobb_41x41x41_uint8.raw and fuel_64x64x64_uint8.raw, and in both cases the contour tree and the segmentations are computed, and the part.raw can indeed be opened using paraview.

I had to make the following changes in the code for the above datasets:

1. Update line 32 of **main.cpp** to the following `Grid3D<unsigned char> grid(dimx,dimy,dimz);`

2. call the function in **main()** as follows: 
`exampleProcessing("C:/tmp/data/marschner_lobb_41x41x41_uint8",41,41,41);` or
`exampleProcessing("C:/tmp/data/fuel_64x64x64_uint8",64,64,64);`
Note that the extension ".raw" is not provided when calling `exampleProcessing()`. The problem could be because you also provide the extension?
